### PR TITLE
feat!: crunchy - default

### PR DESCRIPTION
**Pull request by mishraomp**
---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://quickstart-openshift-2133-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://quickstart-openshift-2133-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/quickstart-openshift/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/quickstart-openshift/actions/workflows/merge.yml)

### CODE REVIEW

**Review by DerekRoberts — APPROVED** [review]
Glorious!  :D

File: vault-sync-secrets/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Use the official Ubuntu base image
+FROM ubuntu:latest
```
Comment (DerekRoberts) [inline vault-sync-secrets/Dockerfile:2]: Try a lighter image?  debian-slim, alpine, rhel-ubi?

Comment (mishraomp) [reply vault-sync-secrets/Dockerfile:2]: I will remove this, this should not have been committed, this is all about crunchy , I am sorry

**Comment by mishraomp** [comment]
@jon-funk , please let me know your feedback as well :)

**Comment by mishraomp** [comment]
closing to test pr close workflow

**Pull request merged**